apps/passport-client/components/screens/EdgeCityScreens/EdgeCityHome.tsx

```diff
@@ -0,0 +1,430 @@
+import { EdDSATicketPCD, EdDSATicketPCDTypeName } from "@pcd/eddsa-ticket-pcd";
+import {
+  EVENT_NAME_TO_TOTAL_SUPPLY,
+  EdgeCityFolderName,
+  HAT_TOKEN_NAME
+} from "@pcd/passport-interface";
+import { useEffect, useState } from "react";
+import styled, { keyframes } from "styled-components";
+import {
+  useFolders,
+  usePCDCollection,
+  usePCDsInFolder
+} from "../../../src/appHooks";
+import { PCDCardList } from "../../shared/PCDCardList";
+import { BalancesTab } from "./BalancesTab";
+import { ExperienceModal } from "./ExperienceModal";
+
+const TABS = [
+  {
+    tab: "ticket",
+    label: "me"
+  },
+  {
+    tab: "experiences",
+    label: "exp"
+  },
+  {
+    tab: "score",
+    label: "bal"
+  }
+] as const;
+type TabId = (typeof TABS)[number]["tab"];
+
+/**
+ * Renders FrogCrypto UI including rendering all EdDSAFrogPCDs.
+ */
+export function EdgeCityHome(): JSX.Element {
+  const edgeCityPCDs = usePCDsInFolder(EdgeCityFolderName);
+  const [tab, setTab] = useState<TabId>("ticket");
+  const [selectedExperience, setSelectedExperience] =
+    useState<EdDSATicketPCD>(null);
+  const pcds = usePCDCollection();
+  // const openInfo = (): void => {
+  //   console.log("hi");
+  // };
+  useEffect(() => {
+    // Set CSS variables on the html element
+    const rootStyle = document.documentElement.style;
+    rootStyle.setProperty("--bg-dark-primary", "black");
+    rootStyle.setProperty("--bg-lite-primary", "black");
+    rootStyle.setProperty("--primary-dark", "black");
+    rootStyle.setProperty("--accent-dark", "white");
+    rootStyle.setProperty("--accent-lite", "white");
+
+    return () => {
+      rootStyle.removeProperty("--bg-dark-primary");
+      rootStyle.removeProperty("--bg-lite-primary");
+      rootStyle.removeProperty("--primary-dark");
+      rootStyle.removeProperty("--accent-dark");
+      rootStyle.removeProperty("--accent-lite");
+      rootStyle.removeProperty("background");
+    };
+  }, []);
+  // TODO: Query param
+  const folders = useFolders(EdgeCityFolderName);
+
+  const pcdsByEventName: Record<string, EdDSATicketPCD[]> = folders
+    .flatMap((folder) => pcds.getAllPCDsInFolder(folder))
+    .filter((pcd): pcd is EdDSATicketPCD => pcd.type === EdDSATicketPCDTypeName)
+    .reduce((acc, pcd) => {
+      // Check if the accumulator already has the eventName key
+      if (!acc[pcd.claim.ticket.eventName]) {
+        // If not, create it and initialize with the current item in an array
+        acc[pcd.claim.ticket.eventName] = [pcd];
+      } else {
+        // If it exists, push the current item to the corresponding array
+        acc[pcd.claim.ticket.eventName].push(pcd);
+      }
+      return acc; // Return the accumulator for the next iteration
```

Comment:
not sure if this is robust - feel like using the event id somehow would be good. probably ok for denver tho.